README.md

```diff
@@ -45,6 +58,17 @@ If you would like to use optimzed bundle of the project you can run t:
 MARLOWE_WEB_SERVER_URL="http://localhost:3780" npm run bundle
 ```
 
+## Building with Nix 
+
+You can build the static site with Nix using:
+```nix 
+nix build .#marlowe-runner
+```
+
+Whenever you make changes to your `spago` packages, you must run the 
+`gen-spago-packages-nix` script (available inside the `nix develop` shell) to 
+recreate `spago-packages.nix`.
```

Comment:
This is somewhat painful. 
There appears to be a way to avoid it using [spago2nix_nativeBuildInputs](https://github.com/justinwoo/spago2nix#1-step-workflow-with-flakenix) however that fails with: 
`spago: security: posix_thread: cannot create process`. Seems to be a bug in spago.
Also, it's possible that the build would require the `--impure` flag.

Reply:
There is also:
* https://github.com/purifix/purifix - https://discourse.purescript.org/t/ann-purifix-the-purescript-package-manager-in-nix/3394

* https://github.com/purs-nix/purs-nix - https://discourse.purescript.org/t/ann-purifix-the-purescript-package-manager-in-nix/3394

Maybe we should experiment with these at some point.

Reply:
purifix seems neat. I'll make a taks for myself to test it out and possibily replace spago2nix with it in all relevant repos